dash-renderer/src/AppProvider.react.tsx

```diff
@@ -31,3 +30,5 @@ AppProvider.defaultProps = {
 };
 
 export default AppProvider;
+
+
```

Comment:
Could redistribute these extra trailing newlines to some of the files that have none 😉 Somehow GH's new icon really screams at me...
<img width="170" alt="Screen Shot 2020-06-02 at 9 42 20 AM" src="https://user-images.githubusercontent.com/2678795/83527317-a3a8b580-a4b5-11ea-8bbd-27e5c3641981.png">

Reply:
https://github.com/plotly/dash/pull/1254/commits/10185dd8688baed6cd24f1db9b5e925f79937546